src/lang/modifyAst.ts

```diff
@@ -1275,7 +1218,28 @@ export function setCallInAst(
       if (err(pipe)) {
         return pipe
       }
-      pipe.node.body.push(call)
+
+      if (pipe.node.type === 'PipeExpression') {
+        pipe.node.body.push(call)
+      } else if (pipe.node.type === 'CallExpressionKw') {
+        const expression = getNodeFromPath<ExpressionStatement>(
+          ast,
+          pathIfPipe,
+          'ExpressionStatement'
+        )
+        if (err(expression) || expression.node.type !== 'ExpressionStatement') {
+          return new Error('Could not retrieve ExpressionStatement')
+        }
+
+        expression.node.expression = createPipeExpression([
+          expression.node.expression,
+          call,
+        ])
+      } else {
+        return new Error(
+          'Expected pipeIfPipe to be a PipeExpression or CallExpressionKw'
+        )
+      }
```

Comment:
This adds coverage to this new util function for the specific case of standalone, no variable `shell` call in the related issue, see tests